ios/magnet/ScannerBle.swift

```diff
@@ -9,19 +9,18 @@
 import Foundation
 
 @objc(ScannerBle) class ScannerBle: NSObject, BeaconScannerDelegate {
-  
   var bridge: RCTBridge!
   var scanner: BeaconScanner!
   
   @objc func start() -> Void {
-    NSLog("starting scanner");
-    self.scanner = BeaconScanner();
+    print("starting scanner")
```

Comment:
Can we just keep the NSLog?
http://stackoverflow.com/questions/25951195/swift-print-vs-println-vs-nslog